.github/workflows/create-docker-prerelease.yml

```diff
@@ -0,0 +1,60 @@
+name: Create docker prerelease
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Create docker prerelease
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            ${{ secrets.DOCKER_ORG }}/${{ secrets.DOCKER_REPOSITORY }}
```

Comment:
I think we want to set a tag here. I think the following is the convention:
```
tags: |
  type=edge
```

We will also want to check this tag type doesn't set the latest tag, or we can add the following to be safe:
```
flavor: |
  latest=false # auto is default
```